### DISCUSSION TITLE
2B Beatmap can no longer be completed

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#25166] [Q&A]
[This](https://osu.ppy.sh/beatmapsets/756794#osu/1605148) beatmap can no longer be completed on lazer, but you were able to do it some versions ago (There are scores on the lazer leaderboard).

No Classic mod.

https://github.com/ppy/osu/assets/53081750/b363bc0e-e803-45f0-acec-9aef6a6e4630

https://github.com/ppy/osu/assets/53081750/b0493e3e-5f9a-4f8d-80b0-a58de7d831fa

With classic  mod.

https://github.com/ppy/osu/assets/53081750/d12ede94-5ee4-4fe7-8bd9-d5763807dd9b

https://github.com/ppy/osu/assets/53081750/409464f5-fead-4bda-9d12-665c9c5614fd

[runtime.log](https://github.com/ppy/osu/files/13007947/runtime.log)
[performance.log](https://github.com/ppy/osu/files/13007950/performance.log)

**Comment by peppy** [comment]
A starting point would be to figure which release was the last release where it worked.

**Comment by Stoppedpuma** [comment]
This has been an issue since I first started using lazer in 2020, IIRC Mayday has always failed at the end of the map since even 2020 builds.
The same issue exists with other aspire maps like "The Solace of Oblivion" that work fine in stable but don't on lazer. I think this has always been an issue unless there was a fix merged at some point that I missed?

**Reply by Theighlin** [reply]
I don't believe this to be the case, I clearly remember completing the map a couple of times on lazer, and as I said, there are submitted scores on the lazer leaderboard that were not cheated by modifying the beatmap (less than 1 year old)

![image](https://github.com/ppy/osu/assets/53081750/c2693d96-ca8a-4108-931f-1a5f85ffbd51)

I have yet to check when this started happening, since autoplay cannot fail in old builds and won't help me there i'll either edit the map to have just the failing part or use a replay to see when this started, shouldn't be too far back.

**Reply by Stoppedpuma** [reply]
Thanks for the correction, I'll do some testing of my own to try to figure out which release was the last one that worked.

**Reply by Stoppedpuma** [reply]
I've done some testing for versions around the 19 month old mark as those seem to be when people were able to pass the map, none of the builds I tested could pass the map:

2022.523.0 FAIL
2022.319.0 FAIL
2022.314.0 FAIL
2022.306.0 FAIL
2022.226.0 FAIL
2022.217.0 FAIL
2022.216.0 FAIL

I can't test any further as I'm met with a libssl error when attempting to open the game on versions prior to 2022.216.0

**Reply by peppy** [reply]
Thank you for checking back on older releases. Now that we know it's not a recent regression, we can likely leave this as a lower priority for when-we-get-to-it.
